### PR TITLE
[14.0][FIX] product_configurator_sale (Custom Values on Reconfigure)

### PR DESCRIPTION
**Pull request by patrickrwilson**
Applies same fix found here: https://github.com/pledra/odoo-product-configurator/pull/185

### CODE REVIEW

**Comment by patrickrwilson** [comment]
@dreispt can you please review

File: product_configurator_sale/wizard/product_configurator.py [hunk 1]
```diff
@@ -52,3 +52,33 @@ def action_config_done(self):
         else:
             self.order_id.write({"order_line": [(0, 0, values)]})
         return
+
+    @api.model
+    def create(self, vals):
+        if self.env.context.get("default_order_line_id", False):
```
Comment (dreispt) [inline product_configurator_sale/wizard/product_configurator.py:58]: Redundant
```suggestion
        if self.env.context.get("default_order_line_id"):
```

**Review by dreispt — APPROVED** [review]

**Comment by dreispt** [comment]
Can you squash please?

**Comment by dreispt** [comment]
@patrickrwilson Note for future commit messages: don't include the Odoo version "[14.0]", only the type of change "[FIX]". 
I usually try to have an explanatory message that I could send to a user to explain the benefit.
Maybe: `[FIX] product_configurator_sale: keep Custom Values on Reconfigure` ?

**Pull request merged**